config/configtls/testdata/test-cert2.pem

```diff
@@ -0,0 +1,14 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
It would be good to have documented somewhere how you generated this so that we can generate it again at a later point using the same features you used.

Reply:
Rather than committing certs that inevitably come with all manner of edge cases and ways they will break and the need to occasionally regenerate them, I've found it easier to simply generate test certs on the fly, into a tempdir if they need to live on a filesystem somewhere.

Reply:
Most likely https://github.com/open-telemetry/opentelemetry-collector/blob/main/Makefile#L326

Reply:
I'll update that build script so it produces both the certs.